exports/@material-ui/core/4.10.1.json

```diff
@@ -113,5 +113,9 @@
   "./Tooltip": "./esm/Tooltip/Tooltip.js",
   "./Typography": "./esm/Typography/Typography.js",
   "./Unstable_TrapFocus": "./esm/Unstable_TrapFocus/Unstable_TrapFocus.js",
-  "./Zoom": "./esm/Zoom/Zoom.js"
+  "./Zoom": "./esm/Zoom/Zoom.js",
+  "./styles/*": "./styles/*.js",
+  "./utils/*": "./utils/*.js",
+  "./package.json": "./package.json",
+  "./": "./"
```

Comment:
we can't add "./"! That tells the builder that every file is the package is accessible and an entrypoint, and if every file is an entrypoint then there's no optimization to be had.

Reply:
I took that from Preact. I thought it was shallow? i.e. only expose files in root?